repository/staged/targets/registry.npmjs.org/keys.json

```diff
@@ -0,0 +1,26 @@
+{
+    "keys": [
+        {
+            "keyId": "SHA256:jl3bwswu80PjjokCgh0o2w5c2U4LhQAE57gj9cz1kzA",
+            "keyUsage": "npm:signatures",
+            "publicKey": {
+                "rawBytes": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE1Olb3zMAFFxXKHiIkQO5cJ3Yhl5i6UPp+IhuteBJbuHcA5UogKo0EWtlWwW6KSaKoTNEYL7JlCQiVnkhBktUgg==",
+                "keyDetails": "PKIX_ECDSA_P256_SHA_256",
+                "validFor": {
+                    "start": "1999-01-01T00:00:00.000Z"
```

Comment:
Is this intentional? I'm guessing this was an old registry key for package signatures?

Reply:
Yes, it is. The key has been used "since forever" so that was around the time npm became a thing, but old enough to accept any registry signature.

Reply:
Or to be more correct, the key is new, but all packages are signed with that key.